### DISCUSSION TITLE
Problem with loading world with version X4613.

### DISCUSSION THREAD

**Discussion by daan2008k** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1056] [Q&A]
I have a problem with loading world with version X4613. The error code is "-1" and i run forge10.13.4.1614-1.7.10. Did i do something wrong or did i miss something? Can someone help me with this problem?

**Comment by JameH2** [comment]
This means that the game is running on *negative* mode, to fix this, you need to go to hbm.cfg and find the option labled “disablePositiveCoords” and set that to *false*

If that still doesnt work, please email mojang support on how to disable *negative* mode

(Post the crash log first)

**Reply by daan2008k** [reply]
I can't find "disablePositiveCoords". Maybe you typed it wrong, are you sure it's not "enablePositiveCoords" or something else?

**Comment by JameH2** [comment]
Please ignore what i said i was joking
